h2o-automl/src/main/java/hex/schemas/TargetEncoderV3.java

```diff
@@ -0,0 +1,36 @@
+package hex.schemas;
+
+import ai.h2o.automl.targetencoding.TargetEncoderBuilder;
+import ai.h2o.automl.targetencoding.TargetEncoderModel;
+import water.api.API;
+import water.api.schemas3.ModelParametersSchemaV3;
+
+import java.util.List;
+
+public class TargetEncoderV3 extends ModelBuilderSchema<TargetEncoderBuilder, TargetEncoderV3, TargetEncoderV3.TargetEncoderParametersV3> {
+  public static class TargetEncoderParametersV3 extends ModelParametersSchemaV3<TargetEncoderModel.TargetEncoderParameters, TargetEncoderParametersV3> {
+    
+    @API(help = "Is blending used ? True by default.", mapsTo = "_withBlending")
```

Comment:
it would be better to rename the parameters to conform to H2O's naming conventions (eg. `_withBlending` violates the rules).